gatk-launch

```diff
@@ -34,18 +34,17 @@ def main(args):
         if len(args) is 0:
             print("")
             print("Usage:")
-            print(" General format")
-            print("    gatk-launch Tool toolArgs -- sparkArgs")
+            print(" Usage template all tools (runs direct runner when used with a Spark tool)")
```

Comment:
My  2 cents, I would say:

"Usage template for all tools (uses --sparkRunner DIRECT when used with a Spark tool)"

Overall I think it looks much better.